calc/src/App.css

```diff
@@ -0,0 +1,133 @@
+body {
+  background-color: #e5e5e5;
+}
+button {
+  border-radius: 0 0 0 0;
```

Comment:
We can simplify it to border-radius: 0;. If we even need to add this property. Because probably border radius is set as default to 0.